components/FormManual/FormManual.js

```diff
@@ -105,6 +164,51 @@ export default function FormManual({
             </button>
           </>
         )}
+
+        <StyledImageInputWrapper>
+          {(imageUploaded ||
+            (actionMode === "edit" &&
+              currentFlashcard?.imageUrl &&
+              showOriginalImage)) && (
```

Comment:
you can refactor this into a variable:
```js
const showUploadPreview = imageUploaded ||
            (actionMode === "edit" &&
              currentFlashcard?.imageUrl &&
              showOriginalImage))
```
And then use this here instead:

```jsx
{showUploadPreview ? <Image Preview ...> : <Upload Button>}
```

Reply:
I have refactored it into an ImagePreview component.